model/task_history_headers_test.go

```diff
@@ -0,0 +1,162 @@
+package model
+
+import (
+	"testing"
+
+	"github.com/evergreen-ci/evergreen"
+	"github.com/evergreen-ci/evergreen/db"
+	"github.com/evergreen-ci/evergreen/model/build"
+	"github.com/evergreen-ci/evergreen/model/task"
+	. "github.com/smartystreets/goconvey/convey"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestFindUniqueBuildVariantNamesByTask(t *testing.T) {
+	Convey("Should return unique build variants for tasks", t, func() {
```

Comment:
Convey is actually an outdated way of writing go tests; we do stick to asserts whenever possible.
If you want to  split this test into subtests, I suggest this method: https://github.com/evergreen-ci/evergreen/blob/4829a262cd4a60ae42b6a547b2c5fbaca5fb2bff/model/generate_test.go#L921